__tests__/BookmarkButton-test.js

```diff
@@ -4,54 +4,110 @@ import BookmarkButton from '@screens/app/Components/BookmarkButton/BookmarkButto
 import renderer from 'react-test-renderer';
 import {shallow} from 'enzyme';
 import {IconButton} from 'react-native-paper';
+import checkPropTypes from 'check-prop-types';
 
-test('bookmark snapshot test', () => {
-  const tree = renderer.create(<BookmarkButton />).toJSON();
-  expect(tree).toMatchSnapshot();
+var assert = require('assert');
+
+test('like button snapshot test', () => {
+  const treeTrue = renderer
+    .create(<BookmarkButton isBookmarked={true} />)
+    .toJSON();
+  expect(treeTrue).toMatchSnapshot();
+  const treeFalse = renderer
+    .create(<BookmarkButton isBookmarked={false} />)
```

Comment:
Should we pass in values? Or allow it to just throw warnings? 

Or do we show both versions?

Reply:
I'm not sure what you mean by this?

Reply:
So i was just asking if we maintain a snapshot test of no prop-type pushed entered, even though we require one to be. 

Like will it benefit us to have a 'broken' snapshot test?

Reply:
I've added it. It throws a warning